app/presenters/queries/expanded_link_set.rb

```diff
@@ -3,10 +3,11 @@ module Queries
     class ExpandedLinkSet
       attr_reader :draft
 
-      def initialize(content_id:, draft: false, locale: Edition::DEFAULT_LOCALE)
+      def initialize(content_id: nil, locale: nil, edition: nil, draft: false)
```

Comment:
Actually, as per above, it's used elsewhere in the app without an edition, for the `GetExpandedLinks` query.